InstallNotes/MakeDefsLocalExamples/Mac-Catalina-GCC.Make.defs.local

```diff
@@ -0,0 +1,34 @@
+# This make defs file is for Mac OS Catalina
+# It assumes that you have used homebrew to install
+# the relevant packages as indicated for the commands below
+# You should also have obtained up to date Xcode command line tools
+# using 'xcode-select --install'
+DIM             = 3
+DEBUG           = TRUE
+OPT             = TRUE
+PRECISION       = DOUBLE
+USE_64          = TRUE
+# brew install gcc
+CXX             = g++-9 -std=c++14
+FC              = gfortran-9
+# brew install libomp
+OPENMPCC        = TRUE
+# brew install mpich
+MPI             = TRUE
+MPI_DIR         = /usr/local/Cellar/mpich/3.3.2
+MPICXX          = g++-9 -std=c++14 -I${MPI_DIR}/include -lmpi -fopenmp
```

Comment:
As above and also the -fopenmp flag should be automatically loaded from [Chombo/lib/mk/compiler/Make.defs.GNU](https://github.com/GRChombo/Chombo/blob/ad44560a2ebe9c8bc0c4dc7e85806710190ea780/lib/mk/compiler/Make.defs.GNU#L82).
```suggestion
MPICXX          = g++-9 -I${MPI_DIR}/include -lmpi
```